ui/templates/base.html

```diff
@@ -4,7 +4,9 @@
     <meta charset="UTF-8">
     <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" />
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
-    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+    {% load static %}
+    <link rel="shortcut icon" href="{% static 'images/favicon.ico' %}" />
```

Comment:
yeah a new template tag might not be the answer here, but loading the `static` template tag within the actual template strikes me as anti-pattern. i feel like it would be cleaner to auto-load the `static` template tag for all templates (or at least this one). i say this while acknowledging the following:

1. other templating libraries (jsp?) expect certain functionality to be loaded in the actual template file
1. after poking around a little, it seems that django docs think this approach is OK
1. this brings up an explicit vs. implicit argument

i just don't love the idea of template code that doesn't directly influence the markup that gets generated. if you agree with the sentiment, feel free to autoload the static tag, but this is ship-able as-is

👍